awsx/schema.json

```diff
@@ -2842,12 +2842,16 @@
             "liftSingleValueMethodReturns": true,
             "packageReferences": {
                 "Pulumi": "3.*",
-                "Pulumi.Aws": "5.*"
+                "Pulumi.Aws": "5.*",
+                "Pulumi.Docker": "3.*"
             }
         },
         "go": {
             "generateResourceContainerTypes": true,
             "importBasePath": "github.com/pulumi/pulumi-awsx/sdk/go/awsx",
+            "internalDependencies": [
+                "github.com/pulumi/pulumi-docker/sdk/v3/go/docker"
```

Comment:
I think it needs to just point to GitHub.com/Pulumi/Pulumi-docker/sdk as that's where the go.mod sits

Reply:
Ah yup - simplifying now 👍

Reply:
Interesting - let's merge this and then look at fixing upstream. That path that upstream suggests feels very incorrect but that's a job for a different day